4ti2interface/4ti2interface.json

```diff
@@ -0,0 +1,71 @@
+{
+  "AbstractHTML": "The <span class=\"pkgname\">4ti2Interface</span> package provides an interface to 4ti2",
+  "ArchiveFormats": ".tar.gz .zip",
+  "ArchiveURL": "https://github.com/homalg-project/homalg_project/releases/download/4ti2Interface-2020.10-02/4ti2Interface-2020.10-02",
+  "Autoload": false,
+  "AvailabilityTest": null,
+  "Date": "16/10/2020",
+  "Dependencies": {
+    "ExternalConditions": [
+      [
+        "4ti2",
+        "https://4ti2.github.io/"
+      ]
+    ],
+    "GAP": ">=4.7",
+    "NeededOtherPackages": [
+      [
+        "io",
+        ">=4.2"
+      ]
+    ],
+    "SuggestedOtherPackages": [
+      [
+        "AutoDoc",
+        ">=2013.08.22"
+      ]
+    ]
+  },
+  "InstallationPath": "pkg/4ti2Interface-2020.10-02",
+  "IssueTrackerURL": "https://github.com/homalg-project/homalg_project/issues",
+  "Keywords": [],
+  "License": "GPL-2.0-or-later",
+  "PackageDoc": [
+    {
+      "ArchiveURLSubset": [
+        "doc"
+      ],
+      "Autoload": false,
+      "BookName": "4ti2Interface",
+      "HTMLStart": "doc/chap0.html",
+      "LongTitle": "An interface to 4ti2.",
+      "PDFFile": "doc/manual.pdf",
+      "SixFile": "doc/manual.six"
+    }
+  ],
+  "PackageInfoURL": "https://homalg-project.github.io/homalg_project/4ti2Interface/PackageInfo.g",
+  "PackageName": "4ti2Interface",
+  "PackageWWWHome": "https://homalg-project.github.io/homalg_project/4ti2Interface",
+  "Persons": [
+    {
+      "Email": "gutsche@mathematik.uni-siegen.de",
+      "FirstNames": "Sebastian",
+      "Institution": "University of Siegen",
+      "IsAuthor": true,
+      "IsMaintainer": true,
+      "LastName": "Gutsche",
+      "Place": "Siegen",
+      "PostalAddress": "Department Mathematik\nUniversität Siegen\nWalter-Flex-Straße 3\n57072 Siegen\nGermany",
+      "WWWHome": "https://sebasguts.github.io"
+    }
+  ],
+  "README_URL": "https://homalg-project.github.io/homalg_project/4ti2Interface/README.md",
+  "Sha256": "7e9d4f8a4bf0c5df58614ade4b29b04610bc2a79bb94d6741e7e0f22e00dc774",
```

Comment:
Maybe we should call the key `ArchiveSHA256` or so, to make clear what it refers to.

I was thinking that we might want to add further checksums, e.g. a `PackageInfoSHA256` (I'll write more on that elsewhere)